lib/Conversion/MooreToCore/MooreToCore.cpp

```diff
@@ -128,6 +138,30 @@ struct CallOpConversion : public OpConversionPattern<func::CallOp> {
   }
 };
 
+struct UnrealizedConversionCastConversion
```

Comment:
Yeah! Although it might be useful if someone ever were to do partial lowering of the Moore dialect. In that case it can also be handy to have this op around. But I don't see that being a real usecase in the foreseeable future :grin: